src/configuration-store/hybrid.store.ts

```diff
@@ -0,0 +1,59 @@
+import { logger } from '../application-logger';
+
+import { IAsyncStore, IConfigurationStore, ISyncStore } from './configuration-store';
+
+export class HybridConfigurationStore<T> implements IConfigurationStore<T> {
+  servingStore: ISyncStore<T>;
+  persistentStore: IAsyncStore<T> | null;
+
+  constructor(servingStore: ISyncStore<T>, persistentStore: IAsyncStore<T> | null) {
+    this.servingStore = servingStore;
+    this.persistentStore = persistentStore;
+  }
+
+  /**
+   * Initialize the configuration store by loading the entries from the persistent store into the serving store.
+   */
+  async init(): Promise<void> {
+    if (this.persistentStore) {
+      if (!this.persistentStore.isInitialized()) {
+        /**
+         * The initial remote request to the remote API failed
+         * or never happened because we are in the cool down period.
+         *
+         * Shows a log message that the assignments served from the serving store
+         * may be stale.
+         */
+        logger.warn(
+          'Persistent store is not initialized from remote configuration. Serving assignments that may be stale.',
```

Comment:
Minor but I think we are prefixing our log statements with something like `Eppo SDK` (or at least mentioning it in the message) so that the string can be searched to get all eppo-sdk-related logs.